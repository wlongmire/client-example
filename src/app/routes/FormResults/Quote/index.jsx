import React, { Component } from 'react'

import { LinkContainer } from 'react-router-bootstrap'

import { connect } from 'react-redux'
import { ButtonGroup, Button } from 'react-bootstrap'

import ToggleDisplay from 'components/shared/ToggleDisplay'
import { commifyNumber } from 'app/utils/utilities'
import classNames from 'classnames'

import ratingProducts from 'config/RatingProducts'
import config from 'config'

class QuoteBlock extends Component {
  constructor(props) {
    super(props)
    this.state = {}
  }

  render() {
    const {
      title, className, totalPremium, basePremium, terrorismCoverage, additionalCoverage
    } = this.props
    
    const typeList = [
        { title: 'Total Premium', value: totalPremium },
        { title: 'Base Premium', value: basePremium },
        { title: 'Additional Coverage', value: additionalCoverage },
        { title: 'Terrorism Coverage', value: terrorismCoverage }
    ]
    
    const quoteValues = typeList.map((item, idx) => (
      <ToggleDisplay
        key={idx}
        show={item.value && item.value > 0}
        render={() => (
          <div className="premiumNumber">
            {item.title}
            <span>${commifyNumber(item.value || 0)}</span>
          </div>)
        }
      />
    ))

    return (
      <div className={classNames('quoteBlock', className)}>
        <h4>{title}</h4>
        { quoteValues }
      </div>)
  }
}

class Quote extends Component {
  constructor(props) {
    super(props)
    this.state = {}
  }

  render() {
    const { ratings, submission } = this.props

    const rating = ratings[submission.type]
    const ratingProduct = ratingProducts[submission.type]

    const emailStatusMap = {
      LOADING: <div className="emailStatus">
        <img src="https://s3.amazonaws.com/ownersedge-cdn/images/ajax-loader.gif" />
        <p>Emails/Submission Forms Currently Being Processed</p><span>From there, all forms needed will be sent to argo and your inbox.</span>
      </div>,
      ERROR: <div className="emailStatus error">
        <p>There appears to be something wrong.</p>
        <span>Please contact us to complete this transaction.</span>
      </div>,
      SUCCESS: <div className="emailStatus success">
        <img src="https://s3.amazonaws.com/ownersedge-cdn/images/thumbs-up.png" />
        <p>Your submission forms have been successfully processed.</p>
        <span>The approparte forms should appear in your inbox within the next minute. Thank you for using Argo Limited.</span>
      </div>
    }

    const underwriters = config.underwriters.map((uw, idx) => (
      <li key={idx}>{uw.name} – {uw.position} – {uw.location} - {uw.phone}</li>
    ))

    return (
      <div>
        <h3>Instant Pricing Indication:</h3>
        <div className="quoteBlocks">
          <QuoteBlock
            title={ratingProduct.name}
            className={classNames(ratingProduct.type, 'primaryPricing')}
            basePremium={rating.premium}
            totalPremium={rating.totalPremium}
            additionalCoverage={rating.additionalCoverage}
            terrorismCoverage={rating.terrorPremium}
          />

          <ToggleDisplay
            show={rating.excessPremium > 0}
            render={() => (
              <QuoteBlock
                title="Excess"
                className="excess"
                basePremium={rating.excessPremium}
                totalPremium={rating.totalExcessPremium}
                terrorismCoverage={rating.excessTerrorPremium}
              />)
            }
          />

          <ToggleDisplay
            show={submission.type === 'ocp'}
            render={() => (
              <div>
                <QuoteBlock
                  title={"Here is what you would pay with an Owner's Interest Policy"}
                  className="oi upsell"
                  basePremium={ratings.oi.premium}
                  totalPremium={ratings.oi.totalPremium}
                  additionalCoverage={ratings.oi.additionalCoverage}
                  terrorismCoverage={ratings.oi.terrorPremium}
                />
              </div>)
            }
          />
        </div>

        <div className="content">
          <p>One of our following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.</p>
          <ul>{underwriters}</ul>
        </div>

        { emailStatusMap[this.props.emailStatus] }

        <div className="legalText">
          The "pricing indication" is issued as a matter of information only  and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
        </div>


        <ButtonGroup>
          <LinkContainer to="/submissions">
            <Button className="btn"> Return to Submissions</Button>
          </LinkContainer>
        </ButtonGroup>

      </div>
    )
  }
}

export default connect()(Quote)