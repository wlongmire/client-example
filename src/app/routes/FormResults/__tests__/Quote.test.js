/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Quote } from '../Quote'

const oiBundleRatingsTwoProps = {
  '13bf709d-0440-4c0a-ac26-5698afc956f2': {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 0,
    excessTerrorPremium: 0,
    instantQuote: true,
    premium: 4950,
    rate: 0,
    terrorPremium: 248,
    totalExcessPremium: 0,
    totalPremium: 5323
  },
  'c8037b9d-0440-4c0a-ac26-5698afc956f2': {
    additionalCoverage: 250,
    anticipatedProjectLength: 12,
    excessPremium: 0,
    excessTerrorPremium: 0,
    instantQuote: true,
    premium: 6950,
    rate: 0,
    terrorPremium: 248,
    totalExcessPremium: 0,
    totalPremium: 7323
  },
  oi: {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 20000,
    excessTerrorPremium: 1000,
    instantQuote: true,
    premium: 4500,
    rate: 0,
    terrorPremium: 225,
    totalExcessPremium: 21000,
    totalPremium: 4850
  }
}


const oiBundleRatingsOneProp = {
  '13bf709d-0440-4c0a-ac26-5698afc956f2': {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 0,
    excessTerrorPremium: 0,
    instantQuote: true,
    premium: 4950,
    rate: 0,
    terrorPremium: 248,
    totalExcessPremium: 0,
    totalPremium: 5323
  },
  oi: {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 20000,
    excessTerrorPremium: 1000,
    instantQuote: true,
    premium: 4500,
    rate: 0,
    terrorPremium: 225,
    totalExcessPremium: 21000,
    totalPremium: 4850
  }
}

const oiBundleRatingsOneWithExcessProp = {
  '13bf709d-0440-4c0a-ac26-5698afc956f2': {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 10000,
    excessTerrorPremium: 1000,
    instantQuote: true,
    premium: 4950,
    rate: 0,
    terrorPremium: 248,
    totalExcessPremium: 11000,
    totalPremium: 5323
  },
  oi: {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 20000,
    excessTerrorPremium: 1000,
    instantQuote: true,
    premium: 4500,
    rate: 0,
    terrorPremium: 225,
    totalExcessPremium: 21000,
    totalPremium: 4850
  }
}


const ocpRatingsProp = {
  ocp: {
    additionalCoverage: null,
    anticipatedProjectLength: 12,
    excessPremium: null,
    excessTerrorPremium: null,
    instantQuote: true,
    premium: 10720,
    rate: 0.87,
    terrorPremium: 536,
    totalExcessPremium: null,
    totalPremium: 11256
  },
  oi: {
    additionalCoverage: 125,
    anticipatedProjectLength: 12,
    excessPremium: 20000,
    excessTerrorPremium: 1000,
    instantQuote: true,
    premium: 4500,
    rate: 0,
    terrorPremium: 225,
    totalExcessPremium: 21000,
    totalPremium: 4850
  }
}

const submission = {
  type: 'oi',
  clearanceStatus: 'pass'
}

const ocpSubmission = {
  type: 'ocp',
  clearanceStatus: 'pass'
}
const userPropsWithBundles = {
  bundles: [
    {
      basePremiumMulitplier: 0.9,
      quoteScreenMessage: 'RPS discount, contingent on binding with Builder’s Risk Package',
      pricingSummaryContent: 'RPS discount, contingent on binding with Builder’s Risk Package',
      productType: 'oi',
      bindOrderMessage: 'Is the Builders Risk bound with the RPS Package Program?',
      id: 'c8037b9d-0440-4c0a-ac26-5698afc956f2',
      name: 'rps'
    },
    {
      basePremiumMulitplier: 1.1,
      quoteScreenMessage: 'Test discount, contingent on binding with Builder’s Risk Package',
      pricingSummaryContent: 'Test discount, contingent on binding with Builder’s Risk Package',
      productType: 'oi',
      bindOrderMessage: 'Test. Is the Interpol Risk bound with the Bumbling Package Program?',
      id: '13bf709d-0440-4c0a-ac26-5698afc956f2',
      name: 'interpol'
    }
  ],
  subId: 'b4a94152-ef9c-40e6-b3b7-8b38fba6ab96',
  username: 'dummyTest@gmail.com',
  email: 'dummyTest@gmail.com',
  broker: 'test-7fd-b3ff-4fd3-9fc2-e752b9f5b002',
  expiration: 'dummyExpiration'
}

const userPropsWithoutBundles = {
  bundles: [],
  subId: 'b4a94152-ef9c-40e6-b3b7-8b38fba6ab96',
  username: 'dummyTest@gmail.com',
  email: 'dummyTest@gmail.com',
  broker: 'test-7fd-b3ff-4fd3-9fc2-e752b9f5b002',
  expiration: 'dummyExpiration'
}

describe('>>> Quote Component with OCP Rating --- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Quote
        submission={ocpSubmission}
        user={userPropsWithBundles}
        emailStatus={'SUCCESS'}
        ratings={ocpRatingsProp}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})


describe('>>> Quote Component with Two Bundles (with Success Email) --- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Quote
        submission={submission}
        user={userPropsWithBundles}
        emailStatus={'SUCCESS'}
        ratings={oiBundleRatingsTwoProps}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})

describe('>>> Quote Component with One Bundle (with Success Email) --- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Quote
        submission={submission}
        user={userPropsWithBundles}
        emailStatus={'SUCCESS'}
        ratings={oiBundleRatingsOneProp}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})

describe('>>> Quote Component with One Bundle and Excess (with Success Email) --- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Quote
        submission={submission}
        user={userPropsWithBundles}
        emailStatus={'SUCCESS'}
        ratings={oiBundleRatingsOneWithExcessProp}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})

describe('>>> Quote Component with Bundles (with Pending Email) --- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Quote
        submission={submission}
        user={userPropsWithBundles}
        emailStatus={'PENDING'}
        ratings={oiBundleRatingsTwoProps}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})


describe('>>> Quote Component without Bundles --- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Quote
        submission={submission}
        user={userPropsWithoutBundles}
        emailStatus={'SUCCESS'}
        ratings={oiBundleRatingsTwoProps}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})