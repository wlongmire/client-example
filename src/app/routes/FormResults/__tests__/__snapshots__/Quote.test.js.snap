// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Quote Component with Bundles (with Pending Email) --- Snapshot +++capturing Snapshot with 1`] = `
<div>
  <h3>
    Instant Pricing Indication:
  </h3>
  <div
    className="quoteBlocks"
  >
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest (Test discount, contingent on binding with Builder’s Risk Package)"
        totalPremium={5323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={250}
        basePremium={6950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest (RPS discount, contingent on binding with Builder’s Risk Package)"
        totalPremium={7323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4500}
        className="oi primaryPricing"
        terrorismCoverage={225}
        title="Owner's Interest (standard rate)"
        totalPremium={4850}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
  </div>
  <div
    className="content"
  >
    <p>
      One of the following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.
    </p>
    <ul>
      <li>
        Jessica Buelow
         – 
        Supervisor
         – 
        New York
         - 
        212-607-8829
      </li>
    </ul>
  </div>
  <div
    className="legalText"
  >
    The "pricing indication" is issued as a matter of information only and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/submissions"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn"
        disabled={false}
      >
         Return to Submissions
      </Button>
    </LinkContainer>
  </ButtonGroup>
</div>
`;

exports[`>>> Quote Component with OCP Rating --- Snapshot +++capturing Snapshot with 1`] = `
<div>
  <h3>
    Instant Pricing Indication:
  </h3>
  <div
    className="quoteBlocks"
  >
    <div>
      <QuoteBlock
        additionalCoverage={null}
        basePremium={10720}
        className="ocp primaryPricing"
        terrorismCoverage={536}
        title="Owners and Contractors Protective"
        totalPremium={11256}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4500}
        className="ocp upsell"
        terrorismCoverage={225}
        title="Here is what you would pay with an Owner's Interest Policy"
        totalPremium={4850}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
  </div>
  <div
    className="content"
  >
    <p>
      One of the following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.
    </p>
    <ul>
      <li>
        Jessica Buelow
         – 
        Supervisor
         – 
        New York
         - 
        212-607-8829
      </li>
    </ul>
  </div>
  <div
    className="emailStatus success"
  >
    <img
      alt="thumbs up"
      src="https://s3.us-east-2.amazonaws.com/ownersedge-dev-assets/images/thumbs-up.png"
    />
    <p>
      Your submission forms have been successfully processed.
    </p>
    <span>
      The appropriate forms should appear in your inbox within the next minute. Thank you for using Argo Limited.
    </span>
  </div>
  <div
    className="legalText"
  >
    The "pricing indication" is issued as a matter of information only and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/submissions"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn"
        disabled={false}
      >
         Return to Submissions
      </Button>
    </LinkContainer>
  </ButtonGroup>
</div>
`;

exports[`>>> Quote Component with One Bundle (with Success Email) --- Snapshot +++capturing Snapshot with 1`] = `
<div>
  <h3>
    Instant Pricing Indication:
  </h3>
  <div
    className="quoteBlocks"
  >
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest (Test discount, contingent on binding with Builder’s Risk Package)"
        totalPremium={5323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4500}
        className="oi primaryPricing"
        terrorismCoverage={225}
        title="Owner's Interest (standard rate)"
        totalPremium={4850}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
  </div>
  <div
    className="content"
  >
    <p>
      One of the following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.
    </p>
    <ul>
      <li>
        Jessica Buelow
         – 
        Supervisor
         – 
        New York
         - 
        212-607-8829
      </li>
    </ul>
  </div>
  <div
    className="emailStatus success"
  >
    <img
      alt="thumbs up"
      src="https://s3.us-east-2.amazonaws.com/ownersedge-dev-assets/images/thumbs-up.png"
    />
    <p>
      Your submission forms have been successfully processed.
    </p>
    <span>
      The appropriate forms should appear in your inbox within the next minute. Thank you for using Argo Limited.
    </span>
  </div>
  <div
    className="legalText"
  >
    The "pricing indication" is issued as a matter of information only and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/submissions"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn"
        disabled={false}
      >
         Return to Submissions
      </Button>
    </LinkContainer>
  </ButtonGroup>
</div>
`;

exports[`>>> Quote Component with One Bundle and Excess (with Success Email) --- Snapshot +++capturing Snapshot with 1`] = `
<div>
  <h3>
    Instant Pricing Indication:
  </h3>
  <div
    className="quoteBlocks"
  >
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest (Test discount, contingent on binding with Builder’s Risk Package)"
        totalPremium={5323}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4500}
        className="oi primaryPricing"
        terrorismCoverage={225}
        title="Owner's Interest (standard rate)"
        totalPremium={4850}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
  </div>
  <div
    className="content"
  >
    <p>
      One of the following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.
    </p>
    <ul>
      <li>
        Jessica Buelow
         – 
        Supervisor
         – 
        New York
         - 
        212-607-8829
      </li>
    </ul>
  </div>
  <div
    className="emailStatus success"
  >
    <img
      alt="thumbs up"
      src="https://s3.us-east-2.amazonaws.com/ownersedge-dev-assets/images/thumbs-up.png"
    />
    <p>
      Your submission forms have been successfully processed.
    </p>
    <span>
      The appropriate forms should appear in your inbox within the next minute. Thank you for using Argo Limited.
    </span>
  </div>
  <div
    className="legalText"
  >
    The "pricing indication" is issued as a matter of information only and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/submissions"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn"
        disabled={false}
      >
         Return to Submissions
      </Button>
    </LinkContainer>
  </ButtonGroup>
</div>
`;

exports[`>>> Quote Component with Two Bundles (with Success Email) --- Snapshot +++capturing Snapshot with 1`] = `
<div>
  <h3>
    Instant Pricing Indication:
  </h3>
  <div
    className="quoteBlocks"
  >
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest (Test discount, contingent on binding with Builder’s Risk Package)"
        totalPremium={5323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={250}
        basePremium={6950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest (RPS discount, contingent on binding with Builder’s Risk Package)"
        totalPremium={7323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4500}
        className="oi primaryPricing"
        terrorismCoverage={225}
        title="Owner's Interest (standard rate)"
        totalPremium={4850}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
  </div>
  <div
    className="content"
  >
    <p>
      One of the following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.
    </p>
    <ul>
      <li>
        Jessica Buelow
         – 
        Supervisor
         – 
        New York
         - 
        212-607-8829
      </li>
    </ul>
  </div>
  <div
    className="emailStatus success"
  >
    <img
      alt="thumbs up"
      src="https://s3.us-east-2.amazonaws.com/ownersedge-dev-assets/images/thumbs-up.png"
    />
    <p>
      Your submission forms have been successfully processed.
    </p>
    <span>
      The appropriate forms should appear in your inbox within the next minute. Thank you for using Argo Limited.
    </span>
  </div>
  <div
    className="legalText"
  >
    The "pricing indication" is issued as a matter of information only and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/submissions"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn"
        disabled={false}
      >
         Return to Submissions
      </Button>
    </LinkContainer>
  </ButtonGroup>
</div>
`;

exports[`>>> Quote Component without Bundles --- Snapshot +++capturing Snapshot with 1`] = `
<div>
  <h3>
    Instant Pricing Indication:
  </h3>
  <div
    className="quoteBlocks"
  >
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest"
        totalPremium={5323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={250}
        basePremium={6950}
        className="oi primaryPricing"
        terrorismCoverage={248}
        title="Owner's Interest"
        totalPremium={7323}
      />
      <ToggleDisplay
        render={[Function]}
        show={false}
      />
    </div>
    <div>
      <QuoteBlock
        additionalCoverage={125}
        basePremium={4500}
        className="oi primaryPricing"
        terrorismCoverage={225}
        title="Owner's Interest"
        totalPremium={4850}
      />
      <ToggleDisplay
        render={[Function]}
        show={true}
      />
    </div>
  </div>
  <div
    className="content"
  >
    <p>
      One of the following underwriters will be in contact with you to finalize your coverage options and assist you with purchase.
    </p>
    <ul>
      <li>
        Jessica Buelow
         – 
        Supervisor
         – 
        New York
         - 
        212-607-8829
      </li>
    </ul>
  </div>
  <div
    className="emailStatus success"
  >
    <img
      alt="thumbs up"
      src="https://s3.us-east-2.amazonaws.com/ownersedge-dev-assets/images/thumbs-up.png"
    />
    <p>
      Your submission forms have been successfully processed.
    </p>
    <span>
      The appropriate forms should appear in your inbox within the next minute. Thank you for using Argo Limited.
    </span>
  </div>
  <div
    className="legalText"
  >
    The "pricing indication" is issued as a matter of information only and does not confer any rights upon the insured or constitute a contract between Colony Specialty and the authorized representative or producer of the insured or the insured.
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/submissions"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn"
        disabled={false}
      >
         Return to Submissions
      </Button>
    </LinkContainer>
  </ButtonGroup>
</div>
`;
